xbmc/music/MusicDatabase.cpp

```diff
@@ -753,7 +753,7 @@ bool CMusicDatabase::GetSong(int idSong, CSong& song)
     {
       const dbiplus::sql_record* const record = m_pDS.get()->get_sql_record();
 
-      int idSongArtist = record->at(songArtistOffset + artistCredit_idArtist).get_asInt();
+      record->at(songArtistOffset + artistCredit_idArtist).get_asInt();
```

Comment:
So the whole line needs deletion, just removing the assign is leaving an unnecessary call that will no more be seen for proper removal :)